docs/source/reproducers/02-layout.md

```diff
@@ -0,0 +1,97 @@
+# Reproduce CI layout
+The CI Framework allows to reproduce the same layout we're consuming in CI.
+
+While it's not that different from the install_yamls standard layout (one CRC,
+one Compute), the CI layout introduces a third node (ansible controller), and
```

Comment:
A link describing the node would be a good touch.

Reply:
we need a schema imho. @pablintino you had something lately - care to share it so that we may introduce it into the doc? :)

Reply:
@cjeanner the one on page 3? https://docs.google.com/presentation/d/1exQNr8xOc4BxPMPhCAm3Us5W7dEO5p3lhQEnztNCySc/edit?usp=sharing